proto/cellarfees/v1/query.proto

```diff
@@ -28,4 +36,17 @@ message QueryModuleAccountsRequest {}
 
 message QueryModuleAccountsResponse {
   string fees_address = 1;
-}
\ No newline at end of file
+}
+
+message QueryCellarFeePoolRequest {}
+
+message QueryCellarFeePoolResponse {
+  repeated cosmos.base.v1beta1.Coin pool = 1
+      [(gogoproto.castrepeated) = "github.com/cosmos/cosmos-sdk/types.Coins", (gogoproto.nullable) = false];
+}
+
+message QueryScheduledAuctionHeightRequest {}
+
+message QueryScheduledAuctionHeightResponse {
+  uint64 height = 1;
+}
```

Comment:
The `height` is actually just the next time the chain will check for eligible denoms to auction, all of which would happen at the same time if found. This means that all denoms will have the same auction height if they are in the pool but don't have an active auction, and if they already have an active auction they don't need a height query to report on them, so I think it would be redundant?